app/app-www/css/shared.css

```diff
@@ -0,0 +1,165 @@
+/* Shared
+// ===
+//
+// 1. Behold my awesome normalizers! 👷
+// 2. Disable long tap menu callout
+//
+*/
+
+@font-face {
+    font-family: "Droid Kufi";
```

Comment:
Yes, I believe so... Droid Kufi is the font eXtra requested arabic be displayed in... it's going away in an upcoming commit :D